dev-vcs/git-cola/git-cola-3.8.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+DISTUTILS_USE_SETUPTOOLS=no
+inherit distutils-r1 readme.gentoo-r1 virtualx xdg-utils
+
+DESCRIPTION="The highly caffeinated git GUI"
+HOMEPAGE="https://git-cola.github.io/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	$(python_gen_cond_dep '
```

Comment:
Did you mean to use python-single implementation on this ebuild? This gen_cond is not necessary with pure distutils.

Reply:
thanks @juippis, maybe the doc at https://dev.gentoo.org/~mgorny/python-guide/distutils.html#dependencies should be adjusted then, it states 

> Dependencies on Python packages are declared using the same method as the underlying eclass — that is, python-r1 or python-single-r1.

 and also the example uses `python_gen_cond_dep`

Reply:
Yep, `DISTUTILS_SINGLE_IMPL` is not declared here, so it uses the functionality from *python-r1.eclass*. So now you want to be looking at 
https://dev.gentoo.org/~mgorny/python-guide/multi.html#dependencies
instead. The example you linked, is correct when using *python-single-r1.eclass*, or when `DISTUTILS_SINGLE_IMPL` is declared with *distutils-r1.eclass*. That's why I asked what is the intention with this package, is it supposed to work multi-impl, or single-impl only?

Reply:
ok, sorry I had _my version_ (from my overlay) in mind where DISTUTILS_SINGLE_IMPL is set and implied it is also set here.
I think setting this / make it single-impl makes sense for this package...